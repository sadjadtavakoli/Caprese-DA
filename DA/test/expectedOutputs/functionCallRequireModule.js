
#1 function callee-test/inputs/simplemodule.js-1-5 entered from arrowFunction-test/unit_tests/functioncallrequiremodule.js-1-5
#end Execution