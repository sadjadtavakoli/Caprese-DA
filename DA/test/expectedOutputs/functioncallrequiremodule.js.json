{"callee-test/inputs/simplemodule.js-1-5":{"impacted":["t_1"]},"keyMap":{"t_1":"test/unit_tests/functioncallrequiremodule.js","t_5":"test/inputs/simplemodule.js","t_21":"callee-test/inputs/simplemodule.js-1-5"}}